monitor/src/main/java/edu/asu/diging/monitor/core/exceptions/NoEmailRecipientException.java

```diff
@@ -0,0 +1,27 @@
+package edu.asu.diging.monitor.core.exceptions;
+
+public class NoEmailRecipientException extends Exception {
+
+    private static final long serialVersionUID = -3658335972369001233L;
```

Comment:
should start at 1

Reply:
Sorry Julia, I'm not sure I understand. Is this regarding the serialVersionUID?

Reply:
ues